tck/src/main/java/org/eclipse/microprofile/reactive/messaging/tck/health/HealthAssertions.java

```diff
@@ -0,0 +1,179 @@
+/**
+ * Copyright (c) 2021 Contributors to the Eclipse Foundation
+ *
+ * See the NOTICE file(s) distributed with this work for additional
+ * information regarding copyright ownership.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * You may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.eclipse.microprofile.reactive.messaging.tck.health;
+
+import java.io.IOException;
+import java.io.InputStream;
+import java.net.HttpURLConnection;
+import java.net.URL;
+import java.util.List;
+import java.util.stream.Collectors;
+
+import javax.json.Json;
+import javax.json.JsonArray;
+import javax.json.JsonObject;
+import javax.json.JsonReader;
+import javax.json.JsonValue;
+
+import static org.junit.Assert.assertEquals;
+
+/**
+ * Health check response representation with ability to assert it's state.
+ */
+class HealthAssertions {
+
+    private int responseCode;
+    private JsonObject jsonResponse;
+
+    /**
+     * Connect health endpoint and download response to be able to assert it later.
+     *
+     * @param url of the health check endpoint
+     * @return new instance of the health assertions
+     */
+    static HealthAssertions create(String url) {
+        try {
+            HttpURLConnection con = (HttpURLConnection) new URL(url).openConnection();
```

Comment:
Do we have to use HTTP to verify the checks? Doesn't Microprofile Health expose an API (asking the question as I never checked).

Reply:
That would be great, I didn't found other way. MP Health TCKs use same approach